src/org/labkey/workflow/WorkflowController.java

```diff
@@ -694,7 +694,7 @@ public void validateForm(WorkflowTaskForm form, Errors errors)
      * Claims a task for a user, making that user the owner and the assignee.
      */
     @RequiresPermission(ReadPermission.class)
-    public class ClaimTaskAction extends ReadOnlyApiAction<WorkflowTaskForm>
+    public class ClaimTaskAction extends MutatingApiAction<WorkflowTaskForm>
     {
         @Override
         public Object execute(WorkflowTaskForm form, BindException errors) throws Exception
```

Comment:
@labkey-susanh 
Looks like ClaimTaskAction is correctly extending MutatingApiAction, in which case, should it have something more than ReadPermission?